etcd_session_store_test.go

```diff
@@ -0,0 +1,91 @@
+package casstore
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"gopkg.in/cas.v2"
+)
+
+var (
+	store        cas.SessionStore
+	_defaultEtcd = "http://127.0.0.1:2379"
```

Comment:
Better not to add etcd in the unit test.

Reply:
improve in the next PR